services/nublado/values-minikube.yaml

```diff
@@ -0,0 +1,99 @@
+nublado:
+  fqdn: minikube.lsst.codes
+
+  oauth_provider: 'jwt'
+
+  hub:
+    image: 'lsstsqre/sciplat-hub:latest'
+
+  wf:
+    ingress_ui:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-method: GET
+        nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Token
+        nginx.ingress.kubernetes.io/auth-signin: "https://minikube.lsst.codes:31337/login"
+        nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=exec:notebook"
+        nginx.ingress.kubernetes.io/configuration-snippet: |
+          proxy_set_header X-Forwarded-Proto https;
+          proxy_set_header X-Forwarded-Port 443;
+          proxy_set_header X-Forwarded-Path /workflow;
+          auth_request_set $auth_token $upstream_http_x_auth_request_token;
+          proxy_set_header X-Portal-Authorization "Bearer $auth_token";
+          error_page 403 = "/auth/forbidden?scope=exec:notebook";
+
+  lab:
+    restrict_nodes: 'true'
+    image:
+      experimentals: 2
+      dailies: 2
+      cachefile: '/home/jovyan/repo-cache.json'
+
+    resources:
+      nodejs_max_mem: '6144'
+      mb_per_cpu: 2048
+
+  routes:
+    soda: '/api/image/soda'
+    external:
+      instance: 'https://minikube.lsst.codes'
+
+  dask:
+    restrict_nodes: 'true'
+
+  vault_secrets:
+    enabled: true
+    path: secret/k8s_operator/minikube.lsst.codes/nublado
+
+  proxy:
+    ingress:
+      annotations:
+        nginx.ingress.kubernetes.io/auth-request-redirect: $request_uri
+        nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Token
+        nginx.ingress.kubernetes.io/auth-signin: "https://minikube.lsst.codes:31337/login"
+        nginx.ingress.kubernetes.io/auth-url: "http://gafaelfawr-service.gafaelfawr.svc.cluster.local:8080/auth?scope=exec:notebook"
+        nginx.ingress.kubernetes.io/configuration-snippet: |
+          proxy_set_header X-Forwarded-Proto https;
+          proxy_set_header X-Forwarded-Port 443;
+          proxy_set_header X-Forwarded-Path /nb;
+          auth_request_set $auth_token $upstream_http_x_auth_request_token;
+          proxy_set_header X-Portal-Authorization "Bearer $auth_token";
+          error_page 403 = "/auth/forbidden?scope=exec:notebook";
+      host: minikube.lsst.codes
+      tls:
+        cluster_issuer: cert-issuer-letsencrypt-dns
+        secret: nublado-tls-secret
+        tls_host: minikube.lsst.codes
+
+  mountpoints: |
+    [
+      { "disabled": true,
+        "mountpoint": "/example -- if it doesn't start with '/' we add it",
+        "fileserver-host": "omit to use Hub settings",
+        "fileserver-export": "defaults to '/exports<mountpoint>'",
+        "mode": "rw or ro, defaults to 'ro'",
+        "options": "mount options, e.g. 'local_lock=all'"
+      },
```

Comment:
If we're keeping this file, this should be a comment.

Reply:
Sure I can do that, it's just in everywhere, because that's how it was done

Reply:
Yeah, understood this is just copy and paste. I think we can avoid configuring nublado 1 for the new environment at all, hopefully.

Reply:
Yeah, I tried to set up the fileserver and the networking to get the NFS mounting to work last week, but it was a hellish doomscape of problems.  The kubernetes host didn't originally know the DNS of private services, so I fixed that (apparently k8s has a long standing policy of not touching host's resolv.conf), and still I couldn't get the NFS services to mount.  Empty config works though, which is nice so that other people who still want to run nublado 1 (I'm thinking bootcamp and other people until we get nublado 2 done) can spin up a dev instance.  It might also help Adam with his dev since you don't have to push docker images across the internet before you test them.

Reply:
That would be nice.  The internal fileserver has very likely bitrotted.

Reply:
Okay just made this an empty list and that should be fine.